autotest/pyscripts/test_gdal_edit.py

```diff
@@ -309,7 +367,7 @@ def test_gdal_edit_py_6():
 # Test -colorinterp_X
 
 
-def test_gdal_edit_py_7():
+def test_gdal_edit_py_8():
```

Comment:
test_gdal_edit_py_8 must be added to the gdaltest_list array towards bottom of the file

Reply:
Done. Thanks!